CodeGen/Generators/UnitsNetGen/QuantityGenerator.cs

```diff
@@ -696,6 +732,81 @@ private void GenerateLogarithmicArithmeticOperators()
 " );
         }
 
+        private void GenerateRelationalOperators()
```

Comment:
I know, but it would still be an improvement. I would rather add more xmldoc to the remaining code, but not part of this PR.

Private vs public does not matter much in the context of CodeGen, since contributors are the only ones who are going to read this code anyway.

Reply:
Done 👍